### ISSUE TITLE
Installer freezes

### ISSUE THREAD

**Issue by ruskiballer** [yusifsalam/t490-macos#30]
Hi folks,
I got to the install window. On the first try I managed to erase my disk and start the installation process. But it freezed at 33min remaining.
Then I tried it again but it freezed at the erase process. 
Somebody got this problem before?
I have the i5 version and want to install Big Sur because the other versions are not working for me.
OpenCore did not show any information in the log it stopped after the start of the installer was successful.

**Comment by yusifsalam** [comment]
What SSD do you have?

**Comment by ruskiballer** [comment]
> What SSD do you have?

I‘m using the fabric ssd so I assume it is also a WD with 256gb storage.

**Comment by yusifsalam** [comment]
I'm confused now. How are you using Safari if the installer freezes?

**Comment by ruskiballer** [comment]
> I'm confused now. How are you using Safari if the installer freezes?

OpenCore works fine and I got to the macOs install screen (where you select Time Machine Backup, Reinstall Big Sur, ...). There is an option for open Safari.
After your message with the ssd I looked if I have another ssd here and found one. This time I partion the drive with GUID and APFS and the installation went through.
So it seems like the problem is fixed. I'm gonna try the install now with the original ssd.
